### DISCUSSION TITLE
怎么让动画从0-1播放，然后1-0这样循环播放？

### DISCUSSION THREAD

**Discussion by chenbingweb** [galacean/discussion#125] [Engine] [answered]
实现第一个动画从0-1播放，然后1-0这样循环播放？没找到相关属性

**Comment by luzhuang** [comment, marked as answer]
1-0 播放 可以将 state的speed设置-1,  可以创建两个state，然后通过transition连接。正播连接倒播再连正播。参考：https://galacean.antgroup.com/engine/docs/latest/cn/animation-animatorController 中的动画过渡。
